frameworks/cassandra/universe/config.json

```diff
@@ -391,11 +530,6 @@
           "description": "The name of the cluster managed by the Service",
           "default": "cassandra"
         },
-        "jmx_port": {
```

Comment:
that would be a breaking change right? for customers using non-secure JMX? 
@samvantran is that intended for secure JMX in Cassandra?

Reply:
Good catch! Yes, we definitely don't want to introduce breaking changes (unless we tell customers and give them a transition period where we deprecate the config). Let's revert this to what it used to be... Actually, our upgrade test should have told us that the upgrade failed due to breaking configs: https://github.com/mesosphere/dcos-commons/blob/master/frameworks/cassandra/tests/test_sanity.py#L32-L37. Did that happen?